process-engine/bpmn-studio
# ✨Add super admin claim to ProcessEngine-Runtime

## Changes

The Runtime now supports a `can_manage_process_instances` claim, which indicates a super admin. 
These types of admins are always allowed to see any Correlation and ProcessModel of any user.

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/336

PR: #1474

## How to test the changes

Best use our IdentityServer for this

- Start a ProcessInstance as User `alice` (has the super admin claim)
- Start a ProcessInstance as User `bob` (is a normal user)
- Login as `alice` and look at the Correlation list
    - The user can see two process instances
- Login as `bob` and look at the Correlation list
    - The user can only see his own process instance
